packages/analytics-core/test/helpers/default.ts

```diff
@@ -1,21 +1,25 @@
 import { Config } from '@amplitude/analytics-types';
 import { getDefaultConfig } from '../../src/config';
 
-export const useDefaultConfig = (): Config => ({
-  apiKey: API_KEY,
-  transportProvider: {
-    send: () => Promise.resolve(null),
-  },
-  storageProvider: {
-    isEnabled: async () => true,
-    get: async () => undefined,
-    set: async () => undefined,
-    remove: async () => undefined,
-    reset: async () => undefined,
-    getRaw: async () => undefined,
-  },
-  ...getDefaultConfig(),
-});
+export const useDefaultConfig = (): Config => {
+  const defaultConfig = getDefaultConfig();
+  defaultConfig.diagnosticProvider.apiKey = API_KEY;
```

Comment:
Maybe this should be mocked like the other providers below?

Reply:
True. Changed in https://github.com/amplitude/Amplitude-TypeScript/pull/568/commits/860bde26d816642ae744ad8d4d14c214f805d6ae